packages/cli/lib/lib/webpack/transform-config.js

```diff
@@ -107,7 +107,15 @@ module.exports = async function (env, webpackConfig, isServer = false) {
 		);
 	}
 
-	const m = require('esm')(module)(myConfig);
+	let m = require('esm')(module)(myConfig);
+
+	// The line above results in an empty object w/ Jest,
+	// so we need to do the following in order to load it:
+	if (Object.keys(m).length === 0) {
+		try {
+			m = require(myConfig);
+		} catch {}
```

Comment:
So this is just making sure that it's actually empty? (and we don't care about an error in that case)

Reply:
That was the idea, yeah. By this point we've already ensured the file exists so if it's imported in some capacity and still empty, that's either a file with no exports or whatever this issue is with `esm` in our test suite. If a user does provide us with an empty config, that shouldn't be an issue.

The `try ... catch` I believe is unnecessary and I'll remove in a second. Left over fragment that I don't believe will do anything ever.